src/content/docs/alerts-applied-intelligence/applied-intelligence/incident-intelligence/change-applied-intelligence-correlation-logic-decisions.mdx

```diff
@@ -1011,29 +1029,30 @@ You can use the correlation assistant to more quickly analyze [incidents](/docs/
 6. If you like what's been simulated, click **Next**, and then name and describe your decision.
 7. If the simulation result shows too many potential incidents, you may want to choose a different set of attributes and incidents for your decision and run another simulation. [Learn more about simulation](#simulations).
 
-### Attribute analysis [#frequency-tips]
-
-Two types of attribute analysis appear in the UI:
-
-* **Common attributes:** This analysis simply highlights attributes and values that are the exact same between all selected incidents.
-* **Similar attributes:** Similarity analysis uses the Levenshtein algorithm with a distance of 3 to find attributes whose values would be the same if 3 or fewer character changes are performed.
-  * Numerical values as well as single character values are filtered out of the results.
-  * Requires two incidents to be selected. Similarity analysis is not performed when 3 or more incidents are selected.
-
-To create the best decisions, we recommend choosing common attributes that have a lower frequency in your incidents. Here are tips for understanding how choosing low or high frequency attributes affects your decisions:
+<CollapserGroup>
+  <Collapser
+    id="frequency-tips"
+    title="Attribute analysis"
+  >
+    Two types of attribute analysis appear in the UI:
 
-* **Low frequency:** As an example, an attribute with a 0% in the frequency column is likely a unique identifier or an attribute that only recently reported in your data in the last month. Choosing low frequency attributes may correlate few events.
-* **High frequency:** On the other end, an attribute with 100% frequency would be one that is present on all your data. Choosing these attributes would correlate all of your events together.
+    * **Common attributes:** This analysis simply highlights attributes and values that are the exact same between all selected incidents.
+    * **Similar attributes:** Similarity analysis uses the Levenshtein algorithm with a distance of 3 to find attributes whose values would be the same if 3 or fewer character changes are performed.
+      * Numerical values as well as single character values are filtered out of the results.
```

Comment:
these last two bulletpoints seem wrongly indented, unless they're supposed to be sub-bullets of 'similar attributes'